### DISCUSSION TITLE
Get workflow id

### DISCUSSION THREAD

**Discussion by shubhamgoyal1402** [uber/cadence#5890] [Q&A] [answered]
How to get the workflow id while the workflow is running in between the program

**Comment by demirkayaender** [comment]
Could you explain your question a little bit more. Are you trying to get the workflow id within the workflow itself?

**Reply by shubhamgoyal1402** [reply, marked as answer]
Issue resolved. Got the ans by using 	wid := workflow.GetInfo(ctx).WorkflowExecution.ID
